var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -60,6 +60,10 @@ class Openblas(MakefilePackage):
     patch('openblas_icc_openmp.patch', when='%intel@16.0:')
     patch('openblas_icc_fortran.patch', when='%intel@16.0:')
 
+    # Fixes compilation error on POWER8 with GCC 7
+    # https://github.com/xianyi/OpenBLAS/pull/1098
+    patch('power8.patch', when='@0.2.18:0.2.19')
```

Comment:
> The patch was created from an merged official PR so I would say it doesn't conflict with the claim of yours.

This statement is kind of debatable. Think of this counter-example: I could prepare a patch that goes from `0.2.18` to `0.2.19` and apply it whenever you build `0.2.18`. I don't think I can claim I am using `openblas@0.2.18` pristine sources at that point.

I understand that it is better to fix something rather than just saying it doesn't work, but 7k lines are really a lot. That's why I asked other people what they think about it. Imo the best solution would be:
- a new `openblas` release from upstream 
- conflicts for `target=ppc64 %gcc@7.1.0:` for older versions

but that's outside of our direct control. Another thing that could work is to just rely on `openblas@develop`.